### PR TITLE
Add warning for evaluating model without weights nor training

### PR DESCRIPTION
**Pull request by seanpmorgan**
Only question would be whether this should instead raise an error?

### CODE REVIEW

File: armory/utils/config_loading.py [hunk 1]
```diff
@@ -58,6 +61,11 @@ def load_model(model_config):
     )
     if not isinstance(model, Classifier):
         raise TypeError(f"{model} is not an instance of {Classifier}")
+    if weights_file is None and not model_config["fit"]:
```
Comment (davidslater) [inline armory/utils/config_loading.py:64]: Suggestion - make this:
```
if weights_file and not model_config["fit"]:
```
Right now, if weights_file == "", it successfully runs but does not output a warning. See:
https://github.com/twosixlabs/armory/blob/master/armory/baseline_models/keras/mnist.py#L55

**Comment by davidslater** [comment]
I would prefer not to raise an error. Sometimes it's nice to ensure inference runs before training.

File: armory/utils/config_loading.py (hunk 1 above)
Comment (seanpmorgan) [reply armory/utils/config_loading.py]: Thanks. Updated to `if not weights_file and not model_config["fit"]:` is that what you meant?

Comment (davidslater) [reply armory/utils/config_loading.py]: Yes, that's what I meant.

**Pull request merged**